test/new-e2e/tests/containers/eks_test.go

```diff
@@ -40,8 +51,14 @@ func TestEKSSuite(t *testing.T) {
 		"ddtestworkload:deploy": auto.ConfigValue{Value: "true"},
 	}
 
-	_, stackOutput, err := infra.GetStackManager().GetStack(ctx, "eks-cluster", stackConfig, eks.Run, false)
-	require.NoError(t, err)
+	_, stackOutput, err := infra.GetStackManager().GetStackNoDeleteOnFailure(ctx, "eks-cluster", stackConfig, eks.Run, false)
+	if !assert.NoError(t, err) {
+		stackName, err := infra.GetStackManager().GetPulumiStackName("eks-cluster")
```

Comment:
Is this err overriding the err from GetStackNoDeleteOnFailure?

Reply:
Yes, the `err` variable of line 56 hides the one of `GetStackNoDeleteOnFailure` line 54.
But anyhow, the `err` of `GetStackNoDeleteOnFailure` is used only in the `if` at the next line, line 55.

Reply:
The error line 54 is logged line 55 by `assert.NoError`.